web-local/src/main/java/com/bazaarvoice/emodb/local/EmoServiceWithZK.java

```diff
@@ -134,7 +134,7 @@ public static void main(String... args) throws Exception {
 
             setPermissionsFromFiles(permissionsYamls, emoConfigYaml);
         } catch (Throwable t) {
-            t.printStackTrace();
+            LoggerFactory.getLogger("org.apache.zookeeper").error("setPermissionsFromFiles failed", t);
```

Comment:
logger needs to be extracted as the class variable

Reply:
done